.github/workflows/pylint.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
+        sudo apt-get update
```

Comment:
Why is this necessary?

Reply:
GitHub tests often failed during installing external modules.
So I applied the suggested solution in [test](https://github.com/snu-quiqcl/iquip/actions/runs/5330613285/jobs/9657522710).